openfisca_france_local/communes/montpellier/eligibilite_metropole_mutuelle.py

```diff
@@ -0,0 +1,18 @@
+from openfisca_france.model.base import Menage, MONTH, Variable
+
+class montpellier_eligibilite_residence(Variable):
+    value_type = bool
+    entity = Menage
+    definition_period = MONTH
+    label = "Éligibilité résidentielle d'un ménage aux dipositifs de la métropole de Montpellier"
+    reference = "https://encommun.montpellier.fr/articles/2024-12-19-lagence-de-la-mutuelle-communale-de-montpellier-est-ouverte"
+
+    def formula(menage, period):
+        return sum(menage('depcom', period) == code for code in [
```

Comment:
Comme tu as défini `value_type = bool` ce serait plus logique d'utiliser `any()` retournant un booléen à la place de `sum()` qui retourne un entier:

```suggestion
        return any(menage('depcom', period) == code for code in [
```